spanner/deployment/src/main/java/io/quarkiverse/googlecloudservices/spanner/deployment/SpannerDevServiceConfig.java

```diff
@@ -0,0 +1,51 @@
+package io.quarkiverse.googlecloudservices.spanner.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+/**
+ * Configuration group for the Spanner. This class holds all the configuration properties
+ * related to the Google Cloud Spanner service for development environments.
+ * <p>
+ * Here is an example of how to configure these properties:
+ * <p>
+ *
+ * <pre>
+ * quarkus.google.cloud.spanner.devservice.enabled = true
+ * quarkus.google.cloud.spanner.devservice.image-name = gcr.io/cloud-spanner-emulator/emulator:1.4.0 # optional
+ * quarkus.google.cloud.spanner.devservice.emulatorPort = 8085 # optional
+ * </pre>
+ */
+@ConfigMapping(prefix = "quarkus.google.cloud.spanner.devservice")
+@ConfigGroup
+public interface SpannerDevServiceConfig {
+
+    /**
+     * Indicates whether the Spanner service should be enabled or not.
+     * The default value is 'false'.
+     */
+    @WithDefault("false")
+    boolean enabled();
+
+    /**
+     * Sets the Docker image name for the Google Cloud SDK.
+     * This image is used to emulate the Spanner service in the development environment.
+     * The default value is 'gcr.io/cloud-spanner-emulator/emulator:1.5.29'.
+     */
+    @WithDefault("gcr.io/cloud-spanner-emulator/emulator:1.5.29")
```

Comment:
Wouldn't it be better to use latest as the default tag? (so no tag)

Reply:
Pro: easier to stay up to date
Con: builds may be less stable if there are unforeseen changes to the emulator image (although it might be nice to see that)

Latest is used in other places (except the firebase base Node version, but that's because firebase-tools has restrictions on the engine version, so latest won't work there).

Changing it to be more consistent with the other setups